coil-base/src/main/java/coil/ImageLoader.kt

```diff
@@ -43,6 +45,16 @@ interface ImageLoader {
      */
     val defaults: DefaultRequestOptions
 
+    /**
+     * Create a new [LoadRequestBuilder].
+     */
+    fun load(context: Context) = LoadRequestBuilder(context, this)
+
+    /**
+     * Create a new [GetRequestBuilder].
+     */
+    fun get() = GetRequestBuilder(this)
```

Comment:
These are declared as part of the interface instead of extension functions so they're more discoverable and are easily callable from Java.